crawler/src/main/java/com/brandwatch/minibcr/crawler/services/CrawlerService.java

```diff
@@ -0,0 +1,30 @@
+package com.brandwatch.minibcr.crawler.services;
+
+
+import org.springframework.stereotype.Service;
+
+import com.github.redouane59.twitter.TwitterClient;
```

Comment:
Any reason why you chose this over others? Just curious.

Reply:
It's first on the list in java community libraries recommended by twitter https://developer.twitter.com/en/docs/twitter-api/tools-and-libraries . It's also the least horrible one (documentation wise).